node/src/components/consensus/highway_core/highway.rs

```diff
@@ -689,6 +684,74 @@ pub(crate) mod tests {
         assert!(highway.add_valid_vertex(vv, &mut rng, now).is_empty());
     }
 
+    #[test]
+    fn missing_dependency() -> Result<(), AddUnitError<TestContext>> {
+        let mut state = State::new_test(WEIGHTS, 0);
+        let mut rng = crate::new_rng();
+        let now: Timestamp = 500.into();
+
+        let _ = add_unit!(state, rng, CAROL, 0xC0; N, N, N)?;
+        let _ = add_unit!(state, rng, CAROL, 0xC1; N, N, N)?;
```

Comment:
Why do we need to bind these?  Aren't we just calling them for effect?

Reply:
You're right, I'll remove the `let _ =`.